### DISCUSSION TITLE
Moving from Gtk2 to Gtk3 (by default)

### DISCUSSION THREAD

**Discussion by trizen** [trizen/obmenu-generator#42] [Ideas]
I'm thinking about moving from Gtk2 to the Gtk3 library by default.

Mainly because Gtk2 is old and deprecated. And also the Gtk2 Perl module fails to build on some distros.

Thoughts?

**Comment by gevan17** [comment]
Dear trizen,

I dropped by to just thank you for the best (imo) pipe menu generator. It is a must for me since the day I moved to openbox (a couple of years ago). I agree with Head-on-a-Stick that you should move to Gtk3. I really miss the icons in Debian 11... 

Thanks in advance!

**Comment by trizen** [comment]
Gtk3 is now the default version: https://github.com/trizen/obmenu-generator/commit/894ba46223fbea521853feebde6cb5ef6cebc179

**Comment by gevan17** [comment]
You're simply unbelievable! I was checking the project's page for the past 6-7 months hoping for what has just happened. I already replaced the old version and put back the -i flag. It works as expected. Thanks a million!

**Comment by trizen** [comment]
Support for Gtk3 was available before as well. It was just not enabled by default.

**Comment by gevan17** [comment]
That's right, but on Debian 11 one could not use the -i flag due to the missing perl gtk2 dependency. It turns out that icons help a lot to quickly locate the desired application, even when there are only 4 or 5 applications in a category menu...
